src/server/pbsd_init.c

```diff
@@ -2086,30 +2076,30 @@ call_log_license(struct work_task *ptask)
 
 	/* log the floating license info */
 
-	log_licenses(&usedlicenses);
+	log_licenses(&license_counts.licenses_high_use);
 
 	/* reset values for time periods that have passed */
 
-	usedlicenses.lu_max_hr = 0;
+	license_counts.licenses_high_use.lu_max_hr = 0;
 	ntime = ptask->wt_event;
 	tms = localtime((time_t *)&ntime);
-	if (tms->tm_mday != usedlicenses.lu_day) {
-		usedlicenses.lu_max_day = 0;
-		usedlicenses.lu_day = tms->tm_mday;
+	if (tms->tm_mday != license_counts.licenses_high_use.lu_day) {
```

Comment:
Use a separate function for it.

Reply:
this has been used only once, don't see a point of writing a function for 3-4 statements